Resources/config/services.yml

```diff
@@ -18,6 +18,7 @@ services:
                 debug_file:       "%azine_hybrid_auth_debug_file%"
             store_for_user:   "%azine_hybrid_auth_store_for_user%"
             store_as_cookie:  "%azine_hybrid_auth_store_as_cookie%"
+            expires_in_days:  "%azine_hybrid_auth_expires_in_days%"
```

Comment:
this specifies that an int is passed as parameter to the constructor of AzineHybridAuth. But the current constructor does not take an int as argument.